osf/management/commands/cumulative_plos_metrics.py

```diff
@@ -0,0 +1,157 @@
+import csv
+import io
+from datetime import datetime
+import logging
+import requests
+import tempfile
+
+from django.core.management.base import BaseCommand
+from keen import KeenClient
+from requests_oauthlib import OAuth2
+
+from addons.osfstorage.models import OsfStorageFile
+from framework.celery_tasks import app as celery_app
+from osf.metrics import PreprintView, PreprintDownload
+from osf.models import Guid, Node, OSFUser, Preprint, \
+    Registration, TrashedFile
+from osf.utils.fields import ensure_str
+from website.settings import KEEN, \
+    PLOS_METRICS_BASE_FOLDER, \
+    PLOS_METRICS_OSF_TOKEN, \
+    PLOS_METRICS_INITIAL_FILE_DOWNLOAD_URL
+
+
+DEFAULT_API_VERSION = '2.14'
```

Comment:
why this version? isn't 2.20 the latest?

Reply:
Our other scripts that do remote OSF file ops also specify `2.14`. I'm not certain it matters, but there may have been breaking changes between `2.14` and `2.20`

Reply:
Confirmed `2.20` works as expected